app/src/main/java/com/android/sample/resources/C.kt

```diff
@@ -29,6 +29,7 @@ object C {
     const val ENROLLED_ACTIVITIES = 1
     const val PAST_ACTIVITIES = 2
 
+    const val CARD_IAMGES_SIZE = 120
```

Comment:
Very good for having added a constant in this file

Reply:
Note that there is another constant in this file with the same value (HALF_SCREEN_TEXT_FIELD_PADDING), but I think this is not problematic since they are used in completely different contexts